src/ctr_logging.c

```diff
@@ -614,6 +616,10 @@ static void reopen_k8s_file(void)
 	if (k8s_log_fd < 0)
 		pexitf("Failed to open log file %s", k8s_log_path);
 
+	unlink(k8s_log_path_tmp_old);
+	link(k8s_log_path, k8s_log_path_tmp_old);
+	rename(k8s_log_path_tmp_old, k8s_log_path_old);
```

Comment:
what's this piece?

Reply:
shouldn't be there :/

removed and pushed a new version